apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[catalogue]/variables/[variable].vue

```diff
@@ -153,8 +153,8 @@ useHead({ title: titlePrefix + variable.value.name });
 
         <ContentBlock
           v-if="cohortsWithMapping.length === 0"
-          id="harmonization-details-no-mapping"
-          title="Harmonization"
+          id="harmonisation-details-no-mapping"
```

Comment:
(see my overall comment, but eg is this a field name ("id") rather than just a field label?)